.github/workflows/ci.yml

```
@@ -0,0 +1,39 @@
+name: Code Quality
+
+on:
+    push:
+        branches:
+            - master
+    pull_request:
+
+jobs:
+    code-quality:
+        name: Code Quality
+        runs-on: ubuntu-latest
+
+        strategy:
+            matrix:
+                php: [ '7.2', '7.3', '7.4' ]
+
+        steps:
+            - name: 'Init repository'
+              uses: actions/checkout@v2
+
+            - name: 'Setup PHP'
+              uses: shivammathur/setup-php@v2
+              with:
+                  php-version: ${{ matrix.php }}
+                  extensions: mbstring, intl, gd, xml, dom, json, fileinfo, curl, zip, iconv, swoole
+                  tools: composer
+
+            - name: "Install dependencies"
+              run: composer install
+
+            - name: 'Tests unit'
+              run: composer tests-unit
+
+            - name: 'Tests examples'
+              run: composer tests-examples
+
+            - name: 'Static analysis'
+              run: composer static-analysis
```

Missing the linter test ;)
```suggestion
              run: composer static-analysis

            - name: 'Code style check'
              run: composer code-style-check
```